src/main/resources/config.yml

```diff
@@ -11,15 +11,15 @@ server:
   # The address your server is located at. Please do not include the port.
   address: play.totalfreedom.me
 
-  # Shown at the server list
-  motd: TotalFreedom &8- Minecraft %mcversion%
+  # Shown at the server list - Please note that support for %mcversion% has been withdrawn
+  motd: TotalFreedom &8- The Original All-Op Server!
 
   # Every word in the MOTD will be a different color unless prefixed by a color code
   colorful_motd: true
 
   # All players who show up as owner
   owners:
-    - markbyron
+    - TheMinecraft
```

Comment:
Unless you know who 'TheMinecraft' is, we might as well leave this an empty array.

Reply:
TheMinecraft is Windows aka the owner of TF

Reply:
Exactly what @Focusvity has said.

Reply:
okay!